### DISCUSSION TITLE
GLPI-AGENT : If I change the job name, the agent keeps the old agen name

### DISCUSSION THREAD

**Discussion by steph461** [glpi-project/glpi-agent#562] [Q&A]
### Bug reporting acknowledgment

Yes, I read it

### Professional support

None

### Describe the bug

Hello

If I change the job name, the agent keeps the old job name.
To change the agent name without performing an installation, I go to C:\Program Files\GLPI-Agent\var\glpi-agent.dump and glpi will change new agent name

![image](https://github.com/glpi-project/glpi-agent/assets/68054991/c2ee44fc-6e76-41dd-94a7-3a592791755c)

Will it be possible if there is a change of name the station agent changes his name ? it's a problem that has existed for a long time :)

I change here manually if the computer change of name
![image](https://github.com/glpi-project/glpi-agent/assets/68054991/5c6c1acf-8049-4b2d-8bde-61c8ae478938)

It's not a bug, maybe an improvement?

### To reproduce

1. inventory computer

### Expected behavior

no

### Operating system

Linux

### GLPI Agent version

1.6.1

### GLPI version

10.0.10, Other (See additional context below)

### GLPIInventory plugin or FusionInventory for GLPI plugin version

GLPI Inventory v1.3.3

**Comment by g-bougard** [comment]
Hi @steph461 

first of all, I don't see any issue in your request so you should have opened firstly a Q&A discussion. Then I'll move this issue as such discussion.

What is not clear in your question is why and for what name you want to change the agent name. Also it seems you're talking about editing GLPI-Agent.dump file and the only right way to do that is to use perl, not a text editor as you may just corrupt the file.

One important point is also the agent must not be running as it still can recreate it with its current deviceid name.

Generally speaking, when an agent name is wrong, we just recommend to remove that file and restart the agent so it generates a new name. If you're problem is related to having domain or no in the name you should use the [assetname-support configuration option](https://glpi-agent.readthedocs.io/en/latest/configuration.html#assetname-support).

**Reply by steph461** [reply]
That's exactly what it is , It's recent since 1.5
Thank you for your feedback.
![image](https://github.com/glpi-project/glpi-agent/assets/68054991/cc309c14-62ed-4b87-a865-a658ac2d0005)
